phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/MainWindowView.java

```diff
@@ -74,16 +74,22 @@ protected void setSideBar(SideBar sideBar) {
         this.leftContent.setContent(sideBar);
     }
 
-    public void showRightView(Node nodeToShow) {
+    public void setCenter(Node nodeToShow) {
         this.content.setCenter(nodeToShow);
     }
 
+    public void showDetailsView(Node nodeToShow) { this.content.setRight(nodeToShow); }
```

Comment:
I'm not even sure that we need a close button anymore. You have to try it out yourself but I didn't miss it.